apps/vscode/src/providers/cell/commands.ts

```diff
@@ -145,6 +146,8 @@ abstract class RunCommand {
   }
 
   private async hasExecutorForLanguage(language: string, document: TextDocument, engine: MarkdownEngine) {
+    // TODO: this is incorrect right? `cellExecutorForLanguage` returns a promise, and a promise will always be truthy?
+    // We should have to await it before doing `!!`
     return !!this.cellExecutorForLanguage(language, document, engine);
```

Comment:
Didn't want to make a change here because it will change the logic, so I left a TODO so I don't forget the next time I'm working in this area.